modules/mono/config.py

```diff
@@ -42,13 +42,24 @@ def copy_file(src_dir, dst_dir, name):
     copyfile(src_path, dst_path)
 
 
+def CustomPathIsDirCreate(key, val, env):
```

Comment:
What is this PascalCase prototype (and camelCase variable) doing in the middle of our glorious PEP-8-compliant Python module? :P

Reply:
Ah, I don't know. I guess SCons examples were a negative influence.

Reply:
Renamed to snake_case.